Trimble.Modus.Components/Styles/DarkTheme.xaml

```diff
@@ -72,4 +72,16 @@
   <Color x:Key="BadgeSecondaryBackgroundColor">#6A6E79</Color>
   <Color x:Key="BadgeAlternateTextColor">#252A2E</Color>
   <Color x:Key="BadgeTextColor">#FFFFFF</Color>
+
+  <!--Segmented Control-->
+  <Color x:Key="SegmentSeparatorColor">#6A6E79</Color>
+  <Color x:Key="SegmentItemTextColor">#FFF</Color>
+  <Color x:Key="SegmentItemSelectedTextColor">#FFF</Color>
+  <Color x:Key="SegmentItemSelectedIconTintColor">#FFF</Color>
+  <Color x:Key="SegmentItemIconTintColor">#FFF</Color>
+
```

Comment:
Can you remove these extra spaces that followed in other controls colors as well?